src/App.jsx

```diff
@@ -81,6 +81,14 @@ function App() {
     })
   }, [usersRevivedAsObject, parsedUserEventsAsObject])
 
+  const userDataAsAnArrayFilteredByFaction = useMemo(() => {
+    const factionSearchParam = factionFilterValue
```

Comment:
NIT: Might as well make this:
```js
const factionSearchParam = factionFilterValue.toLocaleUpperCase();
```
That way you don't have to do that later and it makes a bit. more sense as to why this gets it's own assignment.

Reply:
Makes total sense, change made.